app/classifier.py

```diff
@@ -1,36 +1,62 @@
-from operator import itemgetter
+import json
 import logging
 import string
 import time
 import os
 
+from flask import jsonify
+from flask import Blueprint
+from flask import current_app as app
+from flask import request
 import numpy as np
 import tensorflow as tf
 
 from app import embedder
+from app import model_config as mc
 from app import model_fetcher
 
+# The magic ML threshold of values we're confident are relevant enough
+RETAIN_THRESHOLD = 0.4
 
-def fetchVocab(vocab: str):
+classify_bp = Blueprint('classify_bp', __name__)
+
+
+def loadVocab(vocab: str):
     with open(vocab, 'r') as f:
         return f.readlines()
 
 
 class Classifier:
+    """ Classifier may classify a string sequence's topic probability vector.
+
+        Parameters:
+            path_to_bert (str): the URL path to a BERT model e.g.
+                    http://tf.google.com/bert_uncased.
+            path_to_classifier (str): the local path to the dir
+                    containing a saved classifier model and its variables.
+            path_to_vocab (str): the local path to a vocab label file to
```

Comment:
Is there a way to infer this from path_to_classifier? I still think we should store a config file inside each classifier with path_to_vocab and path_to_bert (essentially InConfig and OutConfig?). Maybe add TODO

Reply:
No nice inference option for now. Adding a todo; I like the idea of a per-model config.